ui/src/app/components/runtime-configuration-panel.spec.tsx

```diff
@@ -644,85 +644,86 @@ describe('RuntimeConfigurationPanel', () => {
     'should set runtime preset values in customize panel instead of getRuntime values ' +
       'if configurationType is GeneralAnalysis',
     async () => {
+      const customMachineType = 'n1-standard-16';
+      const customDiskSize = 1000;
       setCurrentRuntime({
         ...runtimeApiStub.runtime,
         status: RuntimeStatus.Deleted,
         configurationType: RuntimeConfigurationType.GeneralAnalysis,
         gceConfig: {
           ...defaultGceConfig(),
-          machineType: 'n1-standard-16',
-          diskSize: 1000,
+          machineType: customMachineType,
+          diskSize: customDiskSize,
         },
         dataprocConfig: null,
       });
 
+      // show that the preset values do not match the existing runtime
+
+      const { machineType, persistentDisk } =
+        runtimePresets.generalAnalysis.runtimeTemplate.gceWithPdConfig;
+
+      expect(customMachineType).not.toEqual(machineType);
+      expect(customDiskSize).not.toEqual(persistentDisk.size);
+
       const wrapper = await component();
-      await mustClickButton(wrapper, 'Customize');
```

Comment:
it's not necessary to click Customize, because we now always open the panel in Customize mode when there's an existing GCE runtime with no PD (because that configuration is outdated and requires customization)

Not sure how this test was passing before.